### PR TITLE
Add data collection to router-perf

### PR DESCRIPTION
**Pull request by nathan-weinberg**
## Type of change

- [ ] Refactor
- [X] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
With Mr. Sandman deprecated we need the `index_data.json` to be generated for our last round of Perf runs with this workload before we fully move to `ingress-perf`

## Related Tickets & Documents

- Related Issue [NETOBSERV-1330](https://issues.redhat.com/browse/NETOBSERV-1330)

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
To come

### CODE REVIEW

File: workloads/router-perf-v2/ingress-performance.sh [hunk 1]
```diff
@@ -69,6 +69,7 @@ if [[ ${ENABLE_SNAPPY_BACKUP} == "true" ]] ; then
  snappy_backup "csv json" "http-perf.yml" "router-perf-v2"
 fi
 
+env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" WORKLOAD="$WORKLOAD" ES_SERVER="$ES_SERVER" ../../utils/index.sh
```
Comment (paigerube14) [inline workloads/router-perf-v2/ingress-performance.sh:72]: there isnt any JOB_START or JOB_END variable here, think JOB_STATUS value can be replaced with ${result}

Comment (nathan-weinberg) [reply workloads/router-perf-v2/ingress-performance.sh:72]: so do i just eliminate those vars and do the replacement and things should work?

Comment (paigerube14) [reply workloads/router-perf-v2/ingress-performance.sh:72]: You'll have to add the creation of those vars in the script to pass to index.sh

Comment (paigerube14) [reply workloads/router-perf-v2/ingress-performance.sh:72]: just like this: 
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/ae79591659203877f620a4fa69bc3013a0433f76/workloads/kube-burner-ocp-wrapper/run.sh#L129 
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/ae79591659203877f620a4fa69bc3013a0433f76/workloads/kube-burner-ocp-wrapper/run.sh#L132

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Pull request merged**